test/distributed/org/apache/cassandra/distributed/impl/Listen.java

```diff
@@ -34,20 +36,31 @@ public Listen(Instance instance)
 
     public Cancel schema(Runnable onChange)
     {
-        final AtomicBoolean cancel = new AtomicBoolean();
+        return start(onChange, instance::schemaVersion);
+    }
+
+    public Cancel liveMembers(Runnable onChange)
+    {
+        return start(onChange, () -> instance.callsOnInstance(() -> Gossiper.instance.getLiveMembers()).call());
```

Comment:
We have `getLiveMemberCount` but not the complete list. I could add that as well, or change the count method to return the whole list and just do the `size` call outside the instance itself if that makes more sense.

Reply:
Remembered why I didn't expose `liveMembers` directly. In `trunk`, `liveMembers` is a `Set<InetAddressAndPort>` whereas in earlier versions it's just `Set<InetAddress>` and I wasn't sure how we'd want to expose it in the cross-version API. Since the `Listen` class just needed the values to compare, it could encapsulate the differing return values and just work.
Also, exposing either class across the isolated class loader boundary would be an issue I think, which means we'd have to duplicate it in the api package of the distributed tests and copy the values out of the version's copy, which gets a bit ugly.